popups/cloud-games/popup.js

```diff
@@ -103,7 +103,7 @@ export default async ({ addon, msg, safeMsg }) => {
         if (this.projects.length === 0 && this.error !== "no-projects") return null;
         if (this.projectsChecked !== this.projects.length) return null;
         const { id, type } = extractUrl(this.selectedTabUrl);
-        if (!id || gameSet.has(`${type}/${id}`)) return null;
+        if (!id || this.projects.some((el) => el.id == id)) return null;
```

Comment:
Now if you have studio, it doesn't ask to add game from that studio (if you had griffpatch games studio it asked for Slither.io even if that game was in studio)